docs/tables/aws_elb_access_log/queries.md

```diff
@@ -0,0 +1,268 @@
+## Activity Examples
+
+### Daily request trends
+
+Count requests per day to identify traffic patterns over time.
+
+```sql
+select
+  strftime(timestamp, '%Y-%m-%d') as request_date,
+  count(*) as request_count
+from
+  aws_elb_access_log
+group by
+  request_date
+order by
+  request_date asc;
+```
+
+### Top 10 clients by request volume
+
+List the top 10 client IP addresses making requests.
+
+```sql
+select
+  client_ip,
+  count(*) as request_count
+from
+  aws_elb_access_log
+group by
+  client_ip
+order by
+  request_count desc
+limit 10;
+```
+
+### Request distribution by target
+
+Analyze how requests are distributed across target instances.
+
+```sql
+select
+  target_ip,
+  count(*) as request_count,
+  avg(target_processing_time) as avg_processing_time
+from
+  aws_elb_access_log
+where
+  target_ip is not null
+group by
+  target_ip
+order by
+  request_count desc;
+```
+
+### HTTP status code distribution
+
+Analyze the distribution of HTTP status codes returned by the load balancer.
+
+```sql
+select
+  elb_status_code,
+  count(*) as response_count,
+  round(count(*) * 100.0 / sum(count(*)) over (), 2) as percentage
+from
+  aws_elb_access_log
+group by
+  elb_status_code
+order by
+  response_count desc;
+```
+
+## Detection Examples
+
+### Failed backend connections
+
+Identify instances where the load balancer couldn't connect to the backend targets.
+
+```sql
+select
+  timestamp,
+  elb,
+  client_ip,
+  target_ip,
+  request,
+  elb_status_code,
+  target_status_code,
+  error_reason
+from
+  aws_elb_access_log
+where
+  target_status_code is null
+  and elb_status_code = 502
+order by
+  timestamp desc;
+```
+
+### SSL cipher vulnerabilities
+
+Detect usage of deprecated or insecure SSL ciphers.
+
+```sql
+select
+  ssl_cipher,
+  ssl_protocol,
+  count(*) as request_count
+from
+  aws_elb_access_log
+where
+  ssl_cipher is not null
+  and ssl_protocol in ('TLSv1.1', 'TLSv1', 'SSLv3', 'SSLv2') -- Insecure protocols (TLSv1.1, TLSv1, SSLv3, SSLv2)
+group by
+  ssl_cipher,
+  ssl_protocol
+order by
+  request_count desc;
+```
+
+### Suspicious user agents
+
+Identify potentially suspicious user agents making requests.
```

Comment:
Why are these considered suspicious?

Reply:
These user agents are commonly associated with automated scripts, web crawlers, or bots rather than legitimate user activity. While not all requests from these agents are necessarily malicious, a high number of requests from them could indicate scraping, probing, or other automated behaviors that warrant further investigation. The threshold of count(*) > 100 helps identify user agents making an unusually high number of requests, which could be a sign of suspicious activity.